api/machine_types.go

```diff
@@ -208,6 +208,12 @@ type MachineConfig struct {
 	Metrics   *MachineMetrics         `json:"metrics"`
 	Schedule  string                  `json:"schedule,omitempty"`
 	Checks    map[string]MachineCheck `json:"checks,omitempty"`
+	Destroy   bool                    `json:"destroy"`
```

Comment:
Sorry, I missed the discussion. In such case you probably considered `DestroyOnSuccess` but honestly I think DestroyOnExit is the correct one accompanied with docs. If we ever have a need for different "onExit" policies, it can be added as another field. It is not clear why we would want to retry in case of errors and/or at what interval to do so either.